pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -57,9 +60,12 @@ func (t *ofTable) UpdateStatus(flowCountDelta int) {
 
 	if flowCountDelta < 0 {
 		t.flowCount -= uint(-flowCountDelta)
+
 	} else {
 		t.flowCount += uint(flowCountDelta)
 	}
+	metrics.OVSTotalFlowCount.Add(float64(flowCountDelta))
```

Comment:
`t.flowCount` might be redundant with `metrics.OVSTotalFlowCount.` I notice `t.flowCount` could be reset to 0 when reconnecting to OVS (flows are not persistent if OVS restarts). Should it do same to track right number?

Reply:
I thought the counter would be updated regardless of whether metrics is enabled or not, it should be true for prometheus Gauge, but I know K8s Gauge makes some lazy initialization and counter might not change if it's not registered. Anyway it should be fine to have separate counters for now.

Reply:
Ok. I feel we should actually define metrics only when metrics are enabled instead of globals. Will take this up in a follow-up patch. Thanks.

Reply:
AFAIK, it's common to declare the metrics as global variables and register them in init function, then when these metrics are imported by a program, the metrics will be registered automatically. What do you think if we follow this way and just let the option to decide whether to expose metrics API.